bpf/lib/nodeport.h

```diff
@@ -513,57 +480,27 @@ int tail_nodeport_ipv6_dsr(struct __ctx_buff *ctx)
 		ret = DROP_INVALID;
 		goto drop_err;
 	}
-
-	ipv6_addr_copy((union v6addr *)&fib_params.l.ipv6_src,
-		       (union v6addr *)&ip6->saddr);
-	ipv6_addr_copy((union v6addr *)&fib_params.l.ipv6_dst,
-		       (union v6addr *)&ip6->daddr);
-
-	ret = fib_lookup(ctx, &fib_params.l, sizeof(fib_params), 0);
-	if (ret != 0) {
```

Comment:
The new helpers now apply `BPF_FIB_LOOKUP_DIRECT`. Is that a concern / something we should document or at least mention in the commit description?

(applies to all the FIB lookups in nodeport.h I believe)

Reply:
Yeah, for those we don't use the fib rules, so it's an optimization. There has been a PR recently with interest to make it configurable, so I told them to rebase on top of this one: https://github.com/cilium/cilium/pull/23658 .